src/main/java/uk/gov/hmcts/darts/transcriptions/service/impl/TranscriptionServiceImpl.java

```diff
@@ -484,7 +484,7 @@ public List<GetTranscriptionWorkflowsResponse> getTranscriptionWorkflows(Integer
 
         var transcriptionWorkflows = transcriptionWorkflowRepository.findByTranscriptionOrderByWorkflowTimestampDesc(transcription.get());
 
-        if (isCurrent) {
+        if (!isNull(isCurrent) && isCurrent) {
```

Comment:
Might be better to use nonNull(...) than !isNull(...)

Reply:
Fair point, change made.